app/resources/cdl/charge_manager.rb

```diff
@@ -9,7 +9,7 @@ class ChargeManager
     include ActionView::Helpers::DateHelper
     attr_reader :resource_id, :eligible_item_service, :change_set_persister
     attr_writer :resource_charge_list
-    delegate :charged_items, :pending_or_active_holds, :active_hold?, :hold?, :expired_holds, to: :resource_charge_list
+    delegate :charged_items, :pending_or_active_holds, :active_hold?, :hold?, :expired_holds, :active_charge?, to: :resource_charge_list
```

Comment:
It seems like you added this and then didn't use it? Is it left over from an amended commit? It doesn't bother me per se but we probably could take it back out.